docs/repo_structure.md

```diff
@@ -1,54 +1,59 @@
+---
+nav_order: 8
+---
+
 # Repository structure
+{: .no_toc }
+
+1. TOC
+{:toc}
 
+## Rpm-ostree source code
 
-### Rpm-ostree source code
 ```
 .
 └─ src
   ├── app                       rpm-ostree CLI application
-  |
   ├── daemon                    rpm-ostree daemon providing D-Bus API
-  |
   ├── lib                       Public library: contains APIs for exploring rpmdb in OSTrees
-  |
   └── libpriv                   Private API shared between app and daemon
-
 ```
 
-### Rust Libraries
+## Rust Libraries
+
 ```
 .
 └─ rust                         Contains rust libraries that rpm-ostree uses─
 ```
 
-### CI
+## CI
+
 ```
 .
 ├── ci                          Contains scripts to install build dependencies and run tests locally
 └── .papr.yml                   PAPR(https://github.com/projectatomic/papr) uses it to define the test environment and test scripts
```

Comment:
That should be `.cci.jenkinsfile` now. Though feel free to leave as a follow-up also!